app/src/main/java/co/kr/bemyplan/ui/purchase/after/adapter/DayAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package co.kr.bemyplan.ui.purchase.after.adapter
+
+import android.annotation.SuppressLint
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import co.kr.bemyplan.R
+import co.kr.bemyplan.data.DailyContents
+import co.kr.bemyplan.databinding.ItemDayButtonBinding
+
+class DayAdapter: RecyclerView.Adapter<DayAdapter.DayViewHolder>() {
+
+    private var dailyContentsList = listOf<DailyContents>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): DayViewHolder {
+        val binding = ItemDayButtonBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent, false
+        )
+
+        return DayViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: DayViewHolder, position: Int) {
+        holder.onBind(dailyContentsList[position])
+    }
+
+    override fun getItemCount() = dailyContentsList.size
+
+    @SuppressLint("NotifyDataSetChanged")
+    fun setItems(items: List<DailyContents>) {
+        dailyContentsList = items
```

Comment:
> 개인적인 생각이지만.. 가변리스트를 정의하고싶다면 MutableList가 더 좋지않은가 하는 생각은 있습니당..!!
만약 MutableList로 정의 한다면 이 부분은
```suggestion
        dailyContentsList.clear()
        dailyContentsList.addAll(items)
```
요렇게 바꿀 수 있겄징👀👀

Reply:
mutableList로 하면 저런 어노테이션도 뜨지 않죠...?